src/providers/VernaProvider.tsx

```diff
@@ -186,32 +197,36 @@ function VernaProvider({
   }
 
   return (
-    <VernaContext.Provider
-      value={{
-        configSchema,
-        formData,
-        fullSchema,
-        fullUiSchema,
-        handleSubmit,
-        isEditor,
-        objectFieldTemplate: { ...defaultObjectFieldTemplate, ...objectFieldTemplate },
-        schema,
-        selectedFormData,
-        selector,
-        setFormData,
-        setFullSchema,
-        setFullUiSchema,
-        setSchema,
-        setSelector,
-        setUiSchema,
-        setWidgets,
-        transformErrors,
-        uiSchema,
-        widgets,
-      }}
-    >
-      {children}
-    </VernaContext.Provider>
+    <IntlProvider defaultLocale={defaultLocale} locale={locale} messages={translations?.[locale]}>
```

Comment:
Mostly with tests in mind, what do you think to make `locale` optional and if it is undefined, we use `defaultLocale` as default value ?

```suggestion
    <IntlProvider defaultLocale={defaultLocale} locale={locale || defaultLocale} messages={translations?.[locale]}>
```